src/sentry/profiles/task.py

```diff
@@ -472,8 +472,9 @@ def symbolicate(
         return symbolicator.process_jvm(
             exceptions=[],
             stacktraces=stacktraces,
-            modules=modules,
+            modules=[{"uuid": m["uuid"], "type": "proguard"} for m in modules],
```

Comment:
Nit: Maybe add the proguard type at the caller https://github.com/getsentry/sentry/blob/07efd749b07d0638153fb4e24349f364eca5aa33/src/sentry/profiles/task.py#L784-L788

Reply:
You're right, it's nicer.